vehicle/autoware_external_cmd_converter/src/node.cpp

```diff
@@ -135,20 +127,19 @@ void ExternalCmdConverterNode::on_external_cmd(const ExternalControlCommand::Con
 
   // Publish ControlCommand
   autoware_control_msgs::msg::Control output;
-  output.stamp = cmd_ptr->stamp;
-  output.lateral.steering_tire_angle = static_cast<float>(cmd_ptr->control.steering_angle);
-  output.lateral.steering_tire_rotation_rate =
-    static_cast<float>(cmd_ptr->control.steering_angle_velocity);
+  output.stamp = pedals->stamp;
+  output.lateral.steering_tire_angle = static_cast<float>(steering->steering_tire_angle);
+  output.lateral.steering_tire_rotation_rate = 0.0;
```

Comment:
Could you explain why we can set `output.lateral.steering_tire_rotation_rate = 0.0;` ? 
I understand [SteeringCommand.msg](https://github.com/autowarefoundation/autoware_adapi_msgs/blob/66783e83c03baaa0d4bebe1ddf16da7d851074f7/autoware_adapi_v1_msgs/manual/msg/SteeringCommand.msg#L2) does not have rotation rate, but the previously we inputed `cmd_ptr->control.steering_angle_velocity`

Reply:
Thanks for the great comment, I had put it in temporary code and forgot about it.

Reply:
Added steering velocity https://github.com/autowarefoundation/autoware_universe/pull/10354/commits/d513047f1532424b41d79592e097fbf140de8e4a